client-js/src/index.ts

```diff
@@ -362,10 +372,17 @@ export class CEP78Client {
     });
 
     if (config.useSessionCode) {
+      if (!args.collectionName)
+        throw new Error("Missing collectionName argument");
```

Comment:
we don't have linting rules I guess, but usually it's a worth adding parenthesis around the if logic.

Reply:
oh yeah, we have and they are pretty strict but this seems to be accepted anyway.